html/icon-fontawesome.html

```diff
@@ -1389,7 +157,6 @@ <h4 class="card-title">Medical Icons</h4>
             <!-- ============================================================== -->
             <!-- footer -->
             <!-- ============================================================== -->
-            <footer class="footer"> © 2020 Adminwrap by <a href="https://www.wrappixel.com/">wrappixel.com</a> </footer>

```

Comment:
Put this code inplace of footer which was earlier at line 1392
 <footer class="footer"> © 2020 JECRC-IT by <a href=""></a> </footer>

Reply:
changes done